AnnService/inc/Core/BKT/Index.h

```diff
@@ -96,6 +96,7 @@ namespace SPTAG
             ~Index() {}
 
             inline SizeType GetNumSamples() const { return m_pSamples.R(); }
+            inline SizeType GetNumDeletedSamples() const { return (SizeType)m_deletedID.Count(); }
```

Comment:
the function is a little confusing, it returns all deleted element count or samples?

Reply:
It returns all deleted element count.

Reply:
then could you change the function name?

Reply:
ok